meson.build

```diff
@@ -83,6 +83,7 @@ compgen_sources = files(
 )
 
 cc = meson.get_compiler('c')
+librt = cc.find_library('rt', required: true)
```

Comment:
I think this should be required: false, as some systems don't have librt (even though I think they all provide a dummy library).